app-office/texmacs/texmacs-1.99.12.recipe

```diff
@@ -0,0 +1,111 @@
+SUMMARY="A scientific editing platform designed to create beautiful technical documents"
+DESCRIPTION="TeXmacs provides a unified and user friendly framework for editing \
+structured documents with different types of content: text, mathematics, graphics, \
+interactive content, slides, etc.\
+Its rendering engine uses high-quality typesetting algorithms so as to produce \
+professionally looking documents, which can either be printed out or presented from \
+a laptop."
+HOMEPAGE="http://www.texmacs.org"
+COPYRIGHT="Free Software Foundation"
+LICENSE="GNU GPL v3"
+
```

Comment:
Please remove this empty line.

Reply:
ok

Reply:
Again: I was following the advanced example here: https://github.com/haikuports/haikuports/wiki/HaikuPorter-BuildRecipes